crates/pop-cli/src/commands/test/contract.rs

```diff
@@ -24,6 +24,9 @@ pub(crate) struct TestContractCommand {
 		help = "Path to the contracts node to run e2e tests [default: none]"
 	)]
 	node: Option<PathBuf>,
+	/// Automatically source the needed binary required without prompting for confirmation.
+	#[clap(short('y'), long)]
+	skip_confirm: bool,
 }
 
 impl TestContractCommand {
```

Comment:
There is a UX bug here that wasnt picked up in Peters PR. The `clear_screen` should be just before the intro and the sleep should be under the warning. Currently you just get a black screen for 3 seconds. so please could you fix?

Also, it would be better to use `Cli.intro("Starting end-to-end tests")` while you are at it, so we no longer need `clear_screen` as well. That also gets us one step closer to UI testing.